src/Components/Timer/Timer.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react'
+import CountdownTimer from './Timer'
+
+describe('Timer', () => {
+    it('Should have the correct content when rendered', () => {
+        const { getByRole, getByText } = render(<CountdownTimer />)
```

Comment:
Nice job changing this up to destructure the screen methods!

Reply:
fo sho. Per our convo earlier, do y'all think it would be worth it to get Eric's opinion on whether he'd rather see unit tests condensed like this or split up, or nah?

Reply:
I'm always down for more opinions/perspectives!! I agree with your sentiment that it can be easier to troubleshoot when they're broken apart. 

I fell into the habit of bundling my "correct content on render" tests together after the structure of our testing lectures/demos last mod.